pkg/controllers/infra/cluster_controller.go

```diff
@@ -45,6 +49,10 @@ import (
 	"kurator.dev/kurator/pkg/infra/util"
 )
 
+var (
+	log = ctrl.Log.WithName("cluster-controller")
```

Comment:
Do we still need L69?

Reply:
L421 use `log` with name.

Reply:
in `SetupWithManager`, there's no context